insights/parsers/pam.py

```diff
@@ -108,8 +108,8 @@ class PamSSHD(PamDConf):
 """
 
 from collections import namedtuple
-from .. import Parser, get_active_lines, parser
-from ..parsers import unsplit_lines
+from insights import Parser, get_active_lines, parser
+from insights.parsers import unsplit_lines, keyword_search
```

Comment:
It seems to me that that's not "consistent" at all - almost all parser tests, for example, use the absolute import path, and several parsers use absolute paths as well.  Since we're no longer in doubt about the project name, it seems to me that there's no need for relative paths, and if we're going to 'tidy up' to be consistent I'd personally rather everyone use absolute paths.

Where do we stand on this generally?